github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -801,6 +809,8 @@ presubmits:
           value: 1.13.8
         - name: FEATURE_GATES
           value: NonRoot
+        - name: NONROOT
```

Comment:
Do we now have a `NONROOT` variable which is picked up in `kubevirt/kubevirt`? If so, It should probably have been named KUBEVIRT_NONROOT.

Reply:
https://github.com/kubevirt/kubevirt/pull/6112 will use it.
I agree `KUBEVIRT_NONROOT` is a better name.
/hold

Reply:
/hold cancel